file_browser/src/file_browser/local/partials/1-elements/_lists.scss

```diff
@@ -0,0 +1,4 @@
+ul {
+    list-style-type: none;
+    margin: 0;
+  }
```

Comment:
Avoid using generic element selectors, unless you're 100% sure all `ul` elements in the whole app are gonna look like this.
Either way this just looks like a reset, which should be handled by your resets library,